README.md

```diff
@@ -426,6 +426,38 @@ Using a file for a query is also supported with optional parameters to use if th
 const result = await sql.file('query.sql', ['Murray', 68])
 ```
 
+### Rows as Streams
+
+Postgres.js supports [`copy ...`](https://www.postgresql.org/docs/14/sql-copy.html) queries, which are exposed as [Node.js streams](https://nodejs.org/dist/latest-v16.x/docs/api/stream.html).
+
+> **NOTE** This is a low-level API which does not provide any type safety. To make this work, you must match your [`copy query` parameters](https://www.postgresql.org/docs/14/sql-copy.html) correctly to your [Node.js stream read or write](https://nodejs.org/dist/latest-v16.x/docs/api/stream.html) code. Ensure [Node.js stream backpressure](https://nodejs.org/en/docs/guides/backpressuring-in-streams/) is handled correctly to avoid memory exhaustion.
+
+#### ```await sql`copy ... from stdin` -> Writable```
+
+```js
+const { pipeline } = require('stream/promises')
+
+// Stream of users with the default tab delimitated cells and new-line delimitated rows
+const userStream = Readable.from([
+  'Murray\t68\n',
+  'Walter\t80\n'
+])
+
+const query = await sql`copy users (name, age) from stdin`.writable()
+await pipeline(userStream, query);
```

Comment:
I'm following up the above docs about node.js stream backpressure with an example of using pipeline. It seems appropriate to me, but please let me know if this doc example should be changed to something else.

Reply:
Looks great 👍